packages/manager/apps/zimbra/src/pages/dashboard/EmailAccounts/AddAndEditEmailAccount.tsx

```diff
@@ -183,14 +189,30 @@ export default function AddAndEditAccount() {
     return touched && !error;
   };
 
-  const handleFormChange = (name: string, value: string) => {
+  const handleFormChange = (
+    name: string,
+    value: string,
+    untouched?: boolean,
+  ) => {
     const newForm: FormTypeInterface = form;
     newForm[name] = {
       value,
-      touched: true,
+      touched: true && (typeof untouched === 'undefined' ? true : untouched),
```

Comment:
Why true && boolean ? True is required?

Reply:
because when I autocomplete the display name field via first and last name, onOdsValueChange get triggered, so I needed a way to cancel the set of touched on the field if the user didnt actually ever focused the field

Reply:
![image](https://github.com/user-attachments/assets/a8e0666d-7e64-4962-a6ca-593fc7a29d31)
more easy to read ;)